lib/components/Editor/CustomExtensions/Emoji/EmojiSuggestion/ExtensionConfig.js

```diff
@@ -79,7 +79,7 @@ const EmojiSuggestion = Node.create({
         ({ chain }) => {
           chain()
             .focus()
-            .insertContent(`${emoji?.native || ""} `)
+            .insertContent(`${emoji?.native || emoji?.skins[0]?.native || ""} `)
```

Comment:
I understand that `emoji.native` is no longer valid. Replace it with
```suggestion
            .insertContent(`${emoji?.skins[0]?.native || ""} `)
```